docs/Configuracion.md

```diff
@@ -0,0 +1,24 @@
+# Documentos de ReparaHorarios
```

Comment:
¿Por qué pones esto otra vez? ¿Qué tiene que ver con este objetivo?
Si son cambios que habéis hecho después del primer PR, simplemente hacedlo sobre main.